CMakeLists.txt

```diff
@@ -1071,10 +1071,12 @@ if(HELICS_ENABLE_PACKAGE_BUILD)
     endif(WIN32)
 
     if(APPLE)
-        set(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}-macOS")
+        set(CPACK_SYSTEM_NAME "macOS-${CMAKE_SYSTEM_PROCESSOR}")
```

Comment:
is CPACK_SYSTEM_NAME automatically used to generate CPACK_PACKAGE_FILE_NAME?

Reply:
Yes.